src/app/core/market/api/bid/bid.service.ts

```diff
@@ -31,7 +35,10 @@ export class BidService {
                   observer.next(true);
                   observer.complete();
                 }
-              }, observer.error);
+              }, (error) => {
+                error = error.includes('unspent') ? errorType.unspent : errorType.broke
```

Comment:
Fix the logic here please.
It will always output 'broke' if it's not the unspent error, there might be more errors that we're not aware of!

```
if(error.includes('unspent')) {
  error unspent
} else if(error.includes('the string to match the broke error')) {
  error broke
} else {
  just send the error message!
}

```

Reply:
@kewde so you believe there will be more errors accept the above ?

Thanks

Reply:
@kewde what will be the message if other than this we need to show ?

thanks